src/Components/App/App.js

```diff
@@ -6,28 +6,50 @@ import Settings from '../Settings/Settings'
 import About from '../About/About'
 import Stats from '../Stats/Stats'
 import PageNotFound from '../PageNotFound/PageNotFound'
-import './App.scss'
 import { enableNotifications } from '../../helpers/notificationHelpers'
-import { ToastContainer } from 'react-toastify';
+import { ToastContainer } from 'react-toastify'
+import './App.scss'
+
 
 function App() {
   enableNotifications()
+
+  const toggleTimerView = (hide) => {
+    const homeContainer = document.getElementById('HomeContainer')
+    if (hide) {
+      homeContainer.classList.add('hidden')
+    } else {
+      homeContainer.classList.remove('hidden')
+    }
```

Comment:
Just a thought that you could easily turn this into a ternary if you want i.e. `hide ? homeContainer.classList.add('hidden') : homeContainer.classList.remove('hidden')`

Reply:
I did it that way at first, but when looking at convention for multi-line ternary operators, I found a thread from airBnB saying that you can certainly do it that way but their convention is to use an if statement for empathy, so I switched it up.

Reply:
https://github.com/airbnb/javascript/issues/1281

Reply:
Interesting! Thanks for the info. I know in my tech interview, my interviewer had wanted me to refactor an if/else block into a ternary, so I'm guessing it's a little bit based on personal/company preference, which does make sense since both ways are equivalent in terms of efficiency

Reply:
Interesting indeed! Do we have a company preference here at Som Timer?

Reply:
It seems like we've done other stuff with ternary expressions! Though I personally find if/else blocks to be more readable. Open to whichever!

Reply:
Interesting discussion here! I haven't used Ternary Operators enough to have an opinion but it's helpful to see y'all conversing about it.